src/persist/src/operators/input.rs

```diff
@@ -113,15 +113,15 @@ mod tests {
     use timely::dataflow::operators::Capture;
 
     use crate::error::Error;
-    use crate::mem::MemPersister;
-    use crate::persister::Persister;
+    use crate::mem::{MemBlob, MemBuffer};
+    use crate::storage::Persister;
     use crate::Id;
 
     use super::*;
 
     #[test]
     fn new_persistent_unordered_input() -> Result<(), Error> {
-        let mut p = MemPersister::new();
+        let mut p = Persister::<MemBuffer, MemBlob>::new(MemBuffer::new(), MemBlob::new())?;
```

Comment:
nit: the new should be able to type this just fine. remove the `::<MemBuffer, MemBlob>`